pootle/checks.py

```diff
@@ -340,6 +340,81 @@ def check_settings(app_configs=None, **kwargs):
                     id="pootle.W019",
                 ))
 
+    try:
+        edit_coef = settings.POOTLE_SCORING_CONF['EDIT_COEF']
+        try:
+            edit_coef + 0.0
+        except TypeError:
+            errors.append(checks.Critical(
+                _("Invalid value for EDIT_COEF in POOTLE_SCORING_CONF."),
+                hint=_("Set a valid value for EDIT_COEF "
+                       "in POOTLE_SCORING_CONF."),
```

Comment:
you haven't count all possible numeric types here, I think. for me `try..except` was easier to use.